### PR TITLE
Minor Balancing and fixes

### PR DESCRIPTION
**Pull request by X0-11**
:cl: XO-11
tweak: gives all items a base cut delay (no more instant cut walls)
tweak: hyposprays with contents different from their spawn-withs will now perform an armour check
tweak: fixes some missing items on the covenant ship
/:cl:

### CODE REVIEW

File: code/modules/halo/weapons/covenant/lmg.dm [hunk 1]
```diff
@@ -10,9 +10,9 @@
 	projectile_type = /obj/item/projectile/bullet/covenant/plasmarepeater
 	slot_flags = 0	//too unwieldy to carry on your back
 	one_hand_penalty = -1
-	burst = 6
+	//burst = 6
 	max_shots =450
-	burst_delay = 1.3
+	//burst_delay = 1
```
Comment (caelaislinn) [inline code/modules/halo/weapons/covenant/lmg.dm:15]: Undocumented balance changes :scream:

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 2]
```diff
@@ -249,7 +249,9 @@
 	H.mob_size = mob_size
 	var/obj/item/organ/internal/stack/s = H.GetLace()
 	for(var/obj/item/organ/organ in H.contents)
-		if(!istype(organ,/obj/item/organ/internal/stack) && ((organ in H.organs) || (organ in H.internal_organs))) //Preserve neural laces / cortical stacl
+		if(istype(organ,/obj/item/organ/internal/stack))
+			continue
+		if((organ in H.organs) || (organ in H.internal_organs)) //Preserve neural laces / cortical stacks
```
Comment (caelaislinn) [inline code/modules/mob/living/carbon/human/species/species.dm:254]: What's this for?

**Review by caelaislinn — APPROVED** [review]

File: code/modules/halo/weapons/covenant/lmg.dm (hunk 1 above)
Comment (X0-11) [reply code/modules/halo/weapons/covenant/lmg.dm:15]: isn't changing anything, these values do nothing

File: code/modules/mob/living/carbon/human/species/species.dm (hunk 2 above)
Comment (X0-11) [reply code/modules/mob/living/carbon/human/species/species.dm:254]: hopefully a fix to some issues with aheal

**Pull request merged**